test/e2e/certificate-available.spec.ts

```diff
@@ -0,0 +1,90 @@
+import { test, expect } from '@playwright/test';
+import dayjs from 'dayjs';
+import { prisma } from '../../app/db.server';
+import { loggedInAsUser } from './utils';
+
+import type { Certificate } from '@prisma/client';
+
+test.describe('Certificate Page', () => {
+  loggedInAsUser();
+
+  test.beforeAll(async () => {
+    const data: Pick<
```

Comment:
Avoid this pattern in TypeScript.  Because you are hoisting this variable outside of the context of the function call below, it becomes untethered from the type information.  If, instead, you define this data as a literal inside the function call, all type information can be inferred:

```ts
// 1. We can't rely on type inference in fn() by doing this, and must manually type it
const foo: ComplexTypeDeclarationHere = { avoid: 'this' };
fn(foo);

// 2. Let fn() infer the type of our data by defining it within the context in which it will be used:
fn({ do: 'this' });
```